guides/V2_Annotation_Message_Action_Identification.md

```diff
@@ -77,7 +77,7 @@ See also [Annotation](../guides/V1_annotations.md) for information on common ann
 | ------------- |:------------- |:------------ |
 | type          | String        | For a focus annotation, the type will be _message-focus_. |
 | lens          | String        | One of _ActionRequest_, _Question_, and _Commitment_. Watson Work may add lenses at any time, and clients should not depend upon only receiving this set of lenses. |
-| category      | String        | Only ActionRequest currently uses _categories_. These categories are _Add_, _Create_, _Delete_, _Modify_, _Schedule_, _Share_, and _View_. Watson Work Services may add categories at any time, and clients should not depend upon only receiving this set of categories.
+| category      | String        | Only ActionRequest currently uses _categories_. These categories are _Schedule_ and _Share_. Watson Work Services may add categories at any time, and clients should not depend upon only receiving this set of categories.
```

Comment:
If my application was working based on the existence of category `Modify`.... it may no longer work as expected.  Seems to be that a higher level note on this change is needed?

Reply:
Thanks. I updated the What's New with some clarification here, and a link to a research blog post about the approach for focus identification.